Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:latest
+
+WORKDIR /install
+RUN wget https://www.dabeaz.com/ply/ply-3.11.tar.gz --no-check-certificate
```

Comment:
I'm not too happy with --no-check-certificate, but I suspect intermediate certificates may be missing from the web server.